replays/replays.lib.php

```diff
@@ -77,17 +77,47 @@ function toID($name) {
 		return preg_replace('/[^a-z0-9]+/','',$name);
 	}
 
-	function search($term, $page = 0, $isPrivate) {
+	function search($p1 = NULL, $p2 = NULL, $format = NULL, $byRating = false, $isPrivate = false, $page = 0) {
```

Comment:
This seems like a significantly less readable API than my previous one?

I guess you could pass search terms by associative array.

Reply:
I wasn't going for readability, I was going for power. But sure

Reply:
Would it make sense to make the old API a wrapper around this function?

Reply:
I'm fine with just dropping support for the old API.